features/user_can_login.feature

```diff
@@ -25,4 +25,10 @@ Feature: Login to mailbox
         When I fill in "Email" with "user@user.com"
         And I fill in "Password" with "notpassword"
         And I click on "Log in"
-        Then I should see "Invalid Email or password"
\ No newline at end of file
+        Then I should see "Invalid Email or password"
+
+    Scenario: Successfully log out of account
```

Comment:
Since this is a new PR, please extract this to a new feature file

Reply:
done